config/settings/test.yml

```diff
@@ -795,7 +795,7 @@ maintenance:
     bucket: ~
     region: ~
     secret_access_key: ~
-  pagerduty_api_token: FAKE
+  pagerduty_api_token: <%= ENV['maintenance__pagerduty_api_token'] || 'FAKE_PAGER_DUTY_API_TOKEN' %>
```

Comment:
Would `ENV['maintenance__pagerduty_api_token']` ever work in this file? If so, is there any risk of it getting printed to the CI output?

Reply:
ohhh got it 👍 

Thanks!